ui/oopsyraidsy/player_state_tracker.ts

```diff
@@ -408,7 +412,7 @@ export class PlayerStateTracker {
     this.collector.OnMistakeObj(timestamp, mistake);
 
     const targetId = mistake.reportId;
-    if (!targetId || !IsPlayerId(targetId))
+    if (targetId === undefined || targetId.length === 0 || !IsPlayerId(targetId))
```

Comment:
```suggestion
    if (targetId === undefined || !IsPlayerId(targetId))
```

`IsPlayerId` covers this.